src/Http/Exception/TooManyRequestsException.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Http\Exception;
+
+use Exception;
+
+/**
+ * Class TooManyRequestsException.
+ *
+ * @author Arman Tuyakbayev <arman.tuyakbayev@rebilly.com>
+ * @version 0.1
+ */
+final class TooManyRequestsException extends ClientException
+{
+    private $retryAfter = '';
```

Comment:
`Retry-After` is obligatory value, and it is time. I do not think it can be empty.